interop/node-version.json

```diff
@@ -1,19 +1,17 @@
 {
-    "id": "node-js-libp2p-head",
-    "containerImageID": "node-js-libp2p-head",
-    "transports": [
-        "tcp",
-        "ws",
-        {
-            "name": "wss",
-            "onlyDial": true
-        }
-    ],
-    "secureChannels": [
-        "noise"
-    ],
-    "muxers": [
-        "mplex",
-        "yamux"
-    ]
-}
\ No newline at end of file
+  "id": "firefox-js-libp2p-head",
+  "containerImageID": "firefox-js-libp2p-head",
+  "transports": [
+    {
+      "name": "webrtc-direct",
```

Comment:
node.js has no webrtc-direct support - is this a mistake?

Reply:
Definitely a mistake 😓